Binary files hr_holidays_updates-v1/hr_holidays_updates/controllers/__pycache__/main.cpython-312.pyc and hr_holidays_updates-v2/hr_holidays_updates/controllers/__pycache__/main.cpython-312.pyc differ
diff -ruN hr_holidays_updates-v1/hr_holidays_updates/static/src/scss/hrmis_leave_frontend.scss hr_holidays_updates-v2/hr_holidays_updates/static/src/scss/hrmis_leave_frontend.scss
--- hr_holidays_updates-v1/hr_holidays_updates/static/src/scss/hrmis_leave_frontend.scss	2026-01-04 13:38:14.993449700 +0500
+++ hr_holidays_updates-v2/hr_holidays_updates/static/src/scss/hrmis_leave_frontend.scss	2026-01-04 13:38:06.221702900 +0500
@@ -7,6 +7,19 @@
   .o_main_navbar {
     display: none !important;
   }
+
+  /* Hide Odoo website "preview/back" controls (purple top-left arrow). */
+  .o_website_preview_top,
+  .o_website_preview,
+  .o_website_preview_top_left,
+  .o_website_preview_top_right,
+  .o_we_website_top_actions,
+  .o_we_toolbar,
+  .o_we_toolbar_wrapper,
+  .o_we_ui,
+  .o_skip_to_content {
+    display: none !important;
+  }
 }
 
 .hrmis-app {
diff -ruN hr_holidays_updates-v1/hr_holidays_updates/views/hrmis_frontend_templates.xml hr_holidays_updates-v2/hr_holidays_updates/views/hrmis_frontend_templates.xml
--- hr_holidays_updates-v1/hr_holidays_updates/views/hrmis_frontend_templates.xml	2026-01-04 13:38:15.026757600 +0500
+++ hr_holidays_updates-v2/hr_holidays_updates/views/hrmis_frontend_templates.xml	2026-01-04 13:38:06.259112000 +0500
@@ -60,7 +60,7 @@
                                t-att-href="current_employee and ('/hrmis/staff/%s' % current_employee.id) or '/hrmis/services'">
                                 User profile
                             </a>
-                            <a t-attf-class="hrmis-nav__item #{'is-active' if active_menu == 'staff' else ''}" href="/hrmis/staff">Search staff</a>
+                            <!-- <a t-attf-class="hrmis-nav__item #{'is-active' if active_menu == 'staff' else ''}" href="/hrmis/staff">Search staff</a> -->
                             <a class="hrmis-nav__item" href="#">Transfer requests</a>
                             <!-- <a class="hrmis-nav__item" href="#">Promotions</a>
                             <a class="hrmis-nav__item" href="#">Vacancies</a> -->
@@ -70,6 +70,10 @@
                                t-att-href="current_employee and ('/hrmis/staff/%s/leave?tab=new' % current_employee.id) or '/hrmis/services'">
                                 Leave requests
                             </a>
+                             <!-- <a t-attf-class="hrmis-nav__item #{'is-active' if active_menu == 'manage_requests' else ''}"
+                               href="/hrmis/manage/requests?tab=leave">
+                                Manage Requests
+                            </a> -->
                            
                           
                         </nav>
@@ -539,6 +543,88 @@
         </t>
     </template>
 
+    <template id="hrmis_manage_requests" name="HRMIS Manage Requests">
+        <t t-call="hr_holidays_updates.hrmis_ui_layout">
+            <div class="hrmis-panel">
+                <div class="hrmis-panel__header hrmis-panel__header--tabs hrmis-panel__header--tabs--manage">
+                    <a t-attf-class="hrmis-tab hrmis-tab--manage #{'is-active' if tab == 'allocation' else ''}"
+                       t-att-href="'/hrmis/manage/requests?tab=allocation'">
+                        Manage Allocation Requests
+                    </a>
+                    <a t-attf-class="hrmis-tab hrmis-tab--manage #{'is-active' if tab == 'leave' else ''}"
+                       t-att-href="'/hrmis/manage/requests?tab=leave'">
+                        Manage Leave Requests
+                    </a>
+                </div>
+
+                <div class="hrmis-panel__body">
+                    <t t-if="tab == 'leave'">
+                        <t t-if="not leaves">
+                            <div class="hrmis-empty">No leave requests pending your action.</div>
+                        </t>
+                        <t t-foreach="range(len(leaves))" t-as="idx0">
+                            <t t-set="lv" t-value="leaves[idx0]"/>
+                            <div class="hrmis-leave-row">
+                                <div class="hrmis-leave-row__cell hrmis-leave-row__sno"><t t-esc="idx0 + 1"/></div>
+                                <div class="hrmis-leave-row__cell">
+                                    <div class="strong">
+                                        <t t-esc="lv.employee_id.name"/>,
+                                        <t t-esc="(('facility_id' in lv.employee_id._fields and lv.employee_id.facility_id and lv.employee_id.facility_id.name) or ('hrmis_facility_id' in lv.employee_id._fields and lv.employee_id.hrmis_facility_id and lv.employee_id.hrmis_facility_id.name) or 'Facility name')"/>
+                                    </div>
+                                    <div class="sub">
+                                        <t t-esc="(('district_id' in lv.employee_id._fields and lv.employee_id.district_id and lv.employee_id.district_id.name) or ('hrmis_district_id' in lv.employee_id._fields and lv.employee_id.hrmis_district_id and lv.employee_id.hrmis_district_id.name) or 'District name')"/>
+                                    </div>
+                                </div>
+                                <div class="hrmis-leave-row__cell"><t t-esc="lv.holiday_status_id.name"/></div>
+                                <div class="hrmis-leave-row__cell">
+                                    <t t-esc="lv.request_date_from"/> to <t t-esc="lv.request_date_to"/>
+                                </div>
+                                <div class="hrmis-leave-row__cell hrmis-leave-row__actions">
+                                    <a class="hrmis-btn hrmis-btn--outline" t-att-href="'/hrmis/leave/%s' % lv.id">View</a>
+                                    <form t-att-action="'/hrmis/leave/%s/approve' % lv.id" method="post">
+                                        <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                                        <button class="hrmis-btn hrmis-btn--primary" type="submit">Approve</button>
+                                    </form>
+                                </div>
+                            </div>
+                        </t>
+                    </t>
+
+                    <t t-if="tab == 'allocation'">
+                        <t t-if="not allocations">
+                            <div class="hrmis-empty">No allocation requests pending your action.</div>
+                        </t>
+                        <t t-foreach="range(len(allocations))" t-as="idx0">
+                            <t t-set="al" t-value="allocations[idx0]"/>
+                            <div class="hrmis-leave-row">
+                                <div class="hrmis-leave-row__cell hrmis-leave-row__sno"><t t-esc="idx0 + 1"/></div>
+                                <div class="hrmis-leave-row__cell">
+                                    <div class="strong">
+                                        <t t-esc="al.employee_id.name"/>,
+                                        <t t-esc="(('facility_id' in al.employee_id._fields and al.employee_id.facility_id and al.employee_id.facility_id.name) or ('hrmis_facility_id' in al.employee_id._fields and al.employee_id.hrmis_facility_id and al.employee_id.hrmis_facility_id.name) or 'Facility name')"/>
+                                    </div>
+                                    <div class="sub">
+                                        <t t-esc="(('district_id' in al.employee_id._fields and al.employee_id.district_id and al.employee_id.district_id.name) or ('hrmis_district_id' in al.employee_id._fields and al.employee_id.hrmis_district_id and al.employee_id.hrmis_district_id.name) or 'District name')"/>
+                                    </div>
+                                </div>
+                                <div class="hrmis-leave-row__cell"><t t-esc="al.holiday_status_id.name"/></div>
+                                <div class="hrmis-leave-row__cell">
+                                    <t t-esc="al.number_of_days"/> day(s)
+                                </div>
+                                <div class="hrmis-leave-row__cell hrmis-leave-row__actions">
+                                    <a class="hrmis-btn hrmis-btn--outline" t-att-href="'/hrmis/allocation/%s' % al.id">View</a>
+                                    <form t-att-action="'/hrmis/allocation/%s/approve' % al.id" method="post">
+                                        <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                                        <button class="hrmis-btn hrmis-btn--primary" type="submit">Approve</button>
+                                    </form>
+                                </div>
+                            </div>
+                        </t>
+                    </t>
+                </div>
+            </div>
+        </t>
+    </template>
  
 
     <template id="hrmis_allocation_view" name="HRMIS Allocation View">
